### ISSUE TITLE
False positive DL3044

### ISSUE THREAD

**Issue by Stikus** [hadolint/hadolint#568]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
```
ENV SAMTOOLS="$SOFT/samtools-$SAMTOOLS_VERSION/bin/samtools" \
    BGZIP="$SOFT/htslib-$SAMTOOLS_VERSION/bin/bgzip"
```
 should work as 
```
ENV SAMTOOLS="$SOFT/samtools-${SAMTOOLS_VERSION}/bin/samtools" \
    BGZIP="$SOFT/htslib-${SAMTOOLS_VERSION}/bin/bgzip"
```

### Actual behavior
```
ENV SAMTOOLS="$SOFT/samtools-$SAMTOOLS_VERSION/bin/samtools" \
    BGZIP="$SOFT/htslib-$SAMTOOLS_VERSION/bin/bgzip"
```
 throws `DL3044` while 
```
ENV SAMTOOLS="$SOFT/samtools-${SAMTOOLS_VERSION}/bin/samtools" \
    BGZIP="$SOFT/htslib-${SAMTOOLS_VERSION}/bin/bgzip"
```
not

**Comment by lorenzo** [comment]
@m-ildefons would you like to take a look at this?
